source/routing/raptor_api.cpp

```diff
@@ -1410,41 +1410,86 @@ void make_response(navitia::PbCreator& pb_creator,
     }
 }
 
+struct IsochroneCommon {
+    bool clockwise;
+    type::GeographicalCoord coord_origin;
+    map_stop_point_duration departures;
+    DateTime init_dt;
+    type::EntryPoint center;
+    DateTime bound;
+    bt::ptime datetime;
+    IsochroneCommon() {}
+    IsochroneCommon(const bool clockwise,
+                    const type::GeographicalCoord& coord_origin,
+                    const map_stop_point_duration& departures,
+                    const DateTime init_dt,
+                    const type::EntryPoint& center,
+                    const DateTime bound,
+                    bt::ptime datetime)
+        : clockwise(clockwise),
+          coord_origin(coord_origin),
+          departures(departures),
+          init_dt(init_dt),
+          center(center),
+          bound(bound),
+          datetime(datetime) {}
+};
+
+static const boost::optional<IsochroneCommon> make_isochrone_common(
+    RAPTOR& raptor,
+    const type::EntryPoint& center,
+    const uint64_t departure_datetime,
+    const double max_duration,
+    uint32_t max_transfers,
```

Comment:
```suggestion
    const uint32_t max_transfers,
```
? Dunno for RAPTOR& raptor and georef::StreetNetwork& worker

Reply:
Can't for RAPTOR& raptor and georef::StreetNetwork& worker :cry: